frontend/utils/distribution/group-income-distribution.js

```diff
@@ -19,7 +40,7 @@ export default function groupIncomeDistribution ({ state, getters, monthstamp, a
       const amount = adjustment + profile[incomeDetailsType]
       currentIncomeDistribution.push({
         name: username,
-        amount: saferFloat(amount)
+        amount: saferFloat(amount) - (adjusted ? totalPaymentsToOrFrom(paymentsFrom, allPayments, username) : 0)
```

Comment:
Noticing a couple issues:

1. Whenever a final or "final-ish" amount is being calculated, that needs to be wrapped in `saferFloat`. Here, `saferFloat` is being undermined because the calculation for the adjustment is subtracted from `amount` outside of `saferFloat`.
2. The adjust-first approach is called such because the adjustment is done beforehand. Below, the code for the adjustment remains in place when it should be moved/removed (that was the adjust-after approach).

Reply:
Point #1 Response: I totally agree. I'll fix this error ASAP

Point #2 Response: I also agree. I'll fix this error ASAP.